src/activities/Elsa.Activities.Rpa.Web/Extensions/SeleniumExtensions.cs

```diff
@@ -0,0 +1,355 @@
+        }
+
+        public static IWebElement GetFirstAvailableElementByJQuery(this IWebDriver driver, string cssSelector)
+        {
+            var elements = driver.ExecuteJavaScript<IEnumerable<IWebElement>>($"return $(\"{cssSelector}\")");
+            return elements.FirstOrDefault(x => x.Displayed && x.Enabled);
+        }
+        [Obsolete("avoid using this method as much as possible. It requires an active Remote Desktop Session to be performed")]
+        public static void NativeClick(this IWebElement element)
+        {
+            var remote = (RemoteWebElement)element;
+            var action = new Actions(remote.WrappedDriver);
+            action.Click(element).Build().Perform();
+        }
+
+        public static void SetText(this IWebElement element, string text)
+        {
+            if (text == null)
+                text = "";
+            text = text.Replace(Environment.NewLine, @"\r\n");
+            text = text.Replace("'", @"\'");
+            text = JavaScriptEncoder.Default.Encode(text);
+            var remote = (RemoteWebElement)element;
+            var field = element.IsInputTag() ? "value" : "innerText";
+            remote.WrappedDriver.ExecuteJavaScript($"arguments[0].{field} = '{text}'", element);
+        }
+        public static void JQuerySetText(this IWebElement element, string text)
+        {
+            text = text.Replace(Environment.NewLine, @"\r\n");
+            text = text.Replace("'", @"\'");
+            text = JavaScriptEncoder.Default.Encode(text);
+            var remote = (RemoteWebElement)element;
+            remote.WrappedDriver.ExecuteJavaScript(
+               $"$(arguments[0]).val('{text}'); " +
+                "$(arguments[0]).trigger('change'); ", element);
+        }
+        public static void SlideRangeConfirm(this IWebElement element, int valMax)
+        {
+            var remote = (RemoteWebElement)element;
+            remote.WrappedDriver.ExecuteJavaScript(
+                "$(arguments[0]).val('" + valMax + "'); " +
+                "$(arguments[0]).scope().moving = true; " +
+                "$(arguments[0]).scope().clicked = true; " +
+                "$(arguments[0]).triggerHandler('change'); " +
+                "$(arguments[0]).triggerHandler('mouseup'); ", element);
+        }
+        public static void SetAttribute(this IWebElement element, string attribute, string value)
+        {
+            if (value == null)
+                value = "";
+            value = value.Replace("'", @"\'");
+            var remote = (RemoteWebElement)element;
+            remote.WrappedDriver.ExecuteJavaScript($"arguments[0].{attribute} = '{value}'", element);
+        }
+
+        public static bool IsInputTag(this IWebElement element)
+        {
+            return element.TagName.ToLower() == "input" || element.TagName.ToLower() == "textarea";
```

Comment:
A small optimization would be to do this instead of doing `ToLower`:

```csharp
return string.Equals(element.TagName, "input", StringComparison.OrdinalIgnoreCase) || string.Equals( element.TagName, "textarea", StringComparison.OrdinalIgnoreCase);
```